spec/feature/users_index_spec.rb

```diff
@@ -25,12 +25,6 @@
     end
   end
 
-  it 'displays the number of posts each user has written' do
```

Comment:
- [ ] You are currently missing the following test:
- Test that asserts you can see the number of posts each user has written.
Kindly add the missing test case